lib/amazonka/src/Network/AWS/Auth.hs

```diff
@@ -112,10 +122,28 @@ envProfile ::
 envProfile = "AWS_PROFILE"
 
 -- | Default region environment variable
-envRegion ::
+envRegionVar ::
```

Comment:
Why did this change? I can't see a name clash that it's avoiding, and it may break downstream clients. Also, none of the other constants naming environment variables are using a `Var` suffix. I think it should change back.

Reply:
There was some code that used `envRegion` and lead to a warning, but I've removed it before creating a PR.
Reverted.
Also I've exported new env variables, same as `envRegion`.